staff/gianluca-bragaglia/app-template/contact.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Contact</title>
+</head>
+<body>
+    <header>
+        <h1>My App</h1>
+    </header>
+    <nav>
+        <li><a href="home.html">Home</a></li>  
+        <li><a href="news.html">News</a></li> 
+        <li><a href="about.html">About</a></li> 
+        <li><a href="contact.html">Contact</a></li> 
+        <li><a href="profile.html">Profile</a></li> 
+    </nav>
+    <main>
+        <h2>Contact Us</h2>
+        <section>
+                <form>
+                    <p>
+                      <label>Name</label><br>
+                      <input type="text" name="last_name" required>
+                    </p>
+                    <p>
+                      <label>Email</label><br>
+                      <input type="email" name="email" required>
+                    </p>
+                    <p>
+                      <label>Message</label><br>
+                      <textarea rows="4" cols="50">
+                        
+                      </textarea>
+                    </p>
+                    <p>
+                      <button>Send Message</button>
+                    </p>
+                </form>
+        </section>                
+    </main>
+    <footer>
+        <address>
+                My App<br/>
```

Comment:
For better semantics, you should use ```<p>```opposed to ```<br>```. Just like in your "Contact us" form.